app/models/spree/tracker.rb

```diff
@@ -1,10 +1,12 @@
 class Spree::Tracker < ActiveRecord::Base
   belongs_to :store
 
+  validates :tracker_type, presence: true
+
   def self.current(store = nil)
```

Comment:
I think we can deprecate `self.current` since there will be multiple currently active trackers, WDYT?

Reply:
done, https://github.com/solidusio-contrib/solidus_trackers/pull/10/commits/66b4108e3cfc0595cad38403e6b2a6c139959cd5#diff-151b7a1ca23474cfb84b18800b5b9957R23